src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/cookie.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Cookie"
+excerpt: "Browser module: Cookie Class"
+---
+
+Cookie class representing a cookie in the [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext).
+
+| Property | Type   | Description                                                                                                                                                                          |
+| -------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
I see many of the rows in this table reference a default value. Should we consider using a table with a `Default` column as it is used in other parts of the documentation?

Reply:
Sure, see https://github.com/grafana/k6-docs/pull/1328/commits/25091d7a77093b079acf52ee9fe776325867d8f6.